modules/ROOT/pages/migration-munit-test-structure.adoc

```diff
@@ -0,0 +1,46 @@
+= Migrating Test Structure (MUnit)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The MUnit test is divided in three scopes. All of them are optional:
+
+[%header,cols="20,80"]
+|===
+|Scope |Description
+
+| Behavior
+| The behavior scope is meant to set all the preconditions before executing the test logic. +
+Mocks and Spies should go in this section.
+
+| Execution
+| The execution scope is meant to have the testing logic which will wait for all processes to finish before executing the next scope.
```

Comment:
Re Execution, is this saying that the scope contains testing logic and that the scope waits for all the tests to finish before the next scope can execute?  (Asking because it currently sounds like it's saying that the testing logic waits for the processes to finish and that the scope actually executes the next scope. I'm guessing that might not be the intent but am not sure.)

Reply:
The execution scope will contain the call to the flows you want to test or an HTTP request for example. Any execution inside of it, even if it's async, this scope will wait for it to be completed and then proceed to execute the validations.